data/darktableconfig.xml.in

```diff
@@ -1735,6 +1735,27 @@
     <shortdescription>height of the tagging dictionary view</shortdescription>
     <longdescription>maximum height the tagging dictionary view will grow to before scrolling</longdescription>
   </dtconfig>
+  <dtconfig dialog="tagging">
+    <name>plugins/lighttable/tagging/confidence</name>
+    <type min="0" max="100">int</type>
+    <default>50</default>
+    <shortdescription>suggested tags level of confidence</shortdescription>
+    <longdescription>level of confidence to include the tag in the suggestions list, 0: all associated tags, 99: 99% matching associated tags, 100: no matching tag to show only recent tags (faster)</longdescription>
+  </dtconfig>
+  <dtconfig dialog="tagging">
+    <name>plugins/lighttable/tagging/nb_recent_tags</name>
+    <type min="-1" max="1000">int</type>
+    <default>20</default>
+    <shortdescription>number of recently attached tags</shortdescription>
+    <longdescription>number of recently attached tags which are included in the suggestions list. The value `-1' clears the memory</longdescription>
```

Comment:
`clear the memory` means that there is an unlimited number of tags proposed?

Reply:
Nope.
If value = -1, the recent list kept in darktablerc is cleared (the memory) and we skip the saving work.
if value = 0, the recent list in darktablerc keeps the last 10 tags. This minimum can be changed.
Otherwise, the recent list in darktablerc keeps twice the requested number but the UI shows only the requested number.

Reply:
> if value = 0, the recent list in darktablerc keeps the last 10 tags.

Don't you really meant `if value = 10`? If not then I'm highly confused :)

BTW, sorry for this but `The value` in long description should have a lower case for the `t`.

Hopefully we'll find an automatic way to move to capitalized UI at some point.

Reply:
> Don't you really meant if value = 10? If not then I'm highly confused :)

Not sure about what is not clear. Sorry if I give explanation which seems obvious for you.

There are two limits but only one setting.
The setting is about the number of tags shown (UI limit) in the suggestions list, `0->1000` plus `-1` which is equivalent to `0` for the UI limit.
The second limit is related to the number of tags we store in darktablerc, which must be longer to be able to replace the tags which are consumed in suggestions list. This limit is:
- 0 is the setting above is `-1`. Nothing is put is darktablerc. If the user changes the setting to something else, he will have to attach tags to fill in this list before being able to see something in the suggestions list.
- 10 is the setting is between 0 and 5. If the darktablerc list is activated, that's the minimum.
- 2 * setting otherwise.

So, if the user change the UI from 0 to whatever value we have these 10 recent tags in darktablerc and we can show them in the UI without having to attach tags in between.

i'll fix the upper case.

Reply:
done.
I've also replaced "clears memory" by "disables the recent list".